guide/12-enrich-data-with-thematic-information/part4_discover_and_enrich_standard_geographies.ipynb

```diff
@@ -0,0 +1,1208 @@
+{
```

Comment:
<p>I don't think we need to continue mentioning the previous example. Some people will not have seen it or won't care, they want to see this method</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1309/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>